### PR TITLE
Bump linters

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Bump linters
Make camelCase rule mandatory

## Code changes

N/A

## User-facing changes

N/A

## Backwards-incompatible changes

N/A

### CODE REVIEW

**Comment by fcollonval** [comment]
The staging folder is touch by the new prettier format; hence the CI failure for that step.

File: jupyterlab/staging/webpack.config.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ const Handlebars = require('handlebars');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 const merge = require('webpack-merge').default;
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
-  .BundleAnalyzerPlugin;
+const BundleAnalyzerPlugin =
```
Comment (jtpio) [inline jupyterlab/staging/webpack.config.js:14]: Isn't this file copied by `update-core-mode`? If so maybe it should be ignored by prettier / ESLint?

Comment (fcollonval) [reply jupyterlab/staging/webpack.config.js:14]: :+1:

File: .eslintignore [hunk 2]
```diff
@@ -19,17 +19,19 @@ docs/api
...
 examples/example.spec.ts
 examples/federated/core_package/index.template.js
+examples/federated/core_package/index.js
+examples/federated/labextensions
+galata/playwright-report
 jupyterlab/chrome-test.js
 jupyterlab/geckodriver
+jupyterlab/staging/yarn.js
+jupyterlab/staging/index.js
+jupyterlab/staging/webpack.config.js
```
Comment (jtpio) [inline .eslintignore:29]: Looks like we also need to add them to `.prettierignore`.

Comment (fcollonval) [reply .eslintignore:29]: Good catch - I updated the PR; remaining CI errors are not related.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**